dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_2_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryAccessTest.java

```diff
@@ -0,0 +1,125 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.logging.internal_fat;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.RemoteFile;
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import io.openliberty.microprofile.telemetry.internal_fat.shared.TelemetryActions;
+
+@RunWith(FATRunner.class)
+public class TelemetryAccessTest extends FATServletClient {
+
+    private static Class<?> c = TelemetryAccessTest.class;
+
+    public static final String APP_NAME = "MpTelemetryLogApp";
+
+    public static final String SERVER_NAME = "TelemetryAccess";
+
+    //This test will run on all mp 2.0 repeats to ensure we have some test coverage on all versions.
+    //I chose this one because TelemetryMessages is core to this bucket
+    @ClassRule
+    public static RepeatTests rt = TelemetryActions.telemetry20Repeats();
+
+    @Server(SERVER_NAME)
+    public static LibertyServer server;
+
+    // Test server configurations
+    public static final String SERVER_XML_ACCESS_SOURCE_FEATURE = "accessServer.xml";
+
+    private static final String[] EXPECTED_FAILURES = { "CWMOT5005W", "SRVE0315E", "SRVE0777E" };
+
+    private static final String ZERO_SPAN_TRACE_ID = "00000000000000000000000000000000 0000000000000000";
```

Comment:
un-used

Reply:
This will be used for other tests in an upcoming PR.